CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 Changelog
 =========
 
+## TBD 
+
+* Removed the bugsnag android gradle plugin due to incompatibility with UE 5.1+. Symbols can now be uploaded via the [BugSnag CLI](https://docs.bugsnag.com/build-integrations/bugsnag-cli/)[#220](https://github.com/bugsnag/bugsnag-unreal/pull/220)
```

Comment:
I think is the better place to link to; it'll be updated before we release.

Reply:
Done 👍